js/pull-requests.js

```diff
@@ -0,0 +1,38 @@
+var getUserId = document.getElementById("usrid");
+getUserId.addEventListener("keyup", function(event) {
+  event.preventDefault();
+  const con = event.target.value.toLowerCase();
+  var url = "https://api.github.com/repos/rarmatei/js-exercises/pulls";
+  fetch(url)
+    .then(resp => resp.json())
+    .then(function(result) {
+      document.getElementById("pull-requests-list").innerText = "";
+      result.forEach(function(element) {
+        // check the position in the string
+        if (element.user.login.toLowerCase().indexOf(con) != -1) {
+          pullReqAndLink(element);
+          // } else {
+          //   getPullRequestElement.style.display = "none";
+        }
+      });
+    });
+});
+
+function pullReqAndLink(element) {
+  const getPullRequestElement = document.getElementById("pull-requests-list");
+  const addAList = document.createElement("li");
+
+  const addAnchor = linkCreation(element);
```

Comment:
I would go with nouns for the variable names and verbs for function names (similar to the standard DOM functions), so in this case I'd say

```
const anchor = createLink(pullRequest);
```

Also, I'd call the function something like `createPullRequestElement` and probably inline the `linkCreation` function here since it would only add two lines to this function, namely:

```
  addAnchor.href = element.html_url;
  addAnchor.text = element.user.login;
```